go.mod

```diff
@@ -10,12 +10,12 @@ require (
 	github.com/aws/aws-sdk-go-v2 v1.2.0
 	github.com/aws/aws-sdk-go-v2/config v1.0.0
 	github.com/aws/aws-sdk-go-v2/service/athena v1.0.0
-	github.com/bstadlbauer/dask-k8s-operator-go-client v0.1.0
 	github.com/coocood/freecache v1.1.1
+	github.com/dask/dask-kubernetes/v2023 v2023.0.0-20230626103304-abd02cd17b26
```

Comment:
I think this is due to `dask-kubernetes` not prefixing their tags with a `v` (i.e. `2023.6.0` vs. `v2023.6.0`)